README.md

```diff
@@ -55,8 +55,19 @@ Create host reservations.
       ip  => '10.0.1.51',
     }
 
+### ipxe chaining
```

Comment:
There really should be a parameters section for this... But the README as a whole needs a rewrite, so I don't consider this a showstopper or anything.